gpkit/nomials/variables.py

```diff
@@ -164,6 +165,14 @@ def __new__(cls, shape, *args, **descr):
             vl[i] = Variable(**descr)
             vl[i].key.arraykey = arraykey
 
+        if descr.pop("newvariable", True):
+            from .. import MODELS, MODELNUMS, NAMEDVARS
```

Comment:
comparing this to the nearly-identical code in `Variable`, this one is missing

```python
            if MODELS and MODELNUMS:
                NAMEDVARS[tuple(MODELS), tuple(MODELNUMS)].append(self)
```
Should that be added, or should `NAMEDVARS` be removed from the import?